package/MDAnalysis/lib/util.py

```diff
@@ -2543,3 +2544,20 @@ def wrapper(self, *args, **kwargs):
                         self._kwargs[key] = arg
         return func(self, *args, **kwargs)
     return wrapper
+
+def is_installed(modulename: str):
+    """Checks if module is installed
+
+    Parameters
+    ----------
+    modulename : str
+        name of the module to be tested
+    ignore_names : Iterable[str]
+        list of module names to ignore when checking -- will return True
+        if modulename is in ignore_names
+    """
+    try:
+        importlib.import_module(modulename)
+    except ImportError:
+        return False
+    return True
```

Comment:
```suggestion
    return importlib.util.find_spec(modulename) is not None
```
This way the module is not actually imported.